core/src/main/kotlin/ch/ergon/dope/resolvable/clause/SelectClause.kt

```diff
@@ -0,0 +1,97 @@
+package ch.ergon.dope.resolvable.clause
+
+import ch.ergon.dope.resolvable.Resolvable
+import ch.ergon.dope.resolvable.expression.Expression
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.resolvable.expression.unaliased.type.Field
+import ch.ergon.dope.resolvable.expression.unaliased.type.toNumberType
+import ch.ergon.dope.resolvable.formatToQueryString
+import ch.ergon.dope.validtype.BooleanType
+import ch.ergon.dope.validtype.NumberType
+import ch.ergon.dope.validtype.StringType
+import ch.ergon.dope.validtype.ValidType
+
+interface IOffsetClause : Clause
```

Comment:
would move all the interfaces into one file and all the classes with the actual query-constant-strings into another

Reply:
done